torch/_dynamo/convert_frame.py

```diff
@@ -1200,9 +1200,17 @@ def __call__(
                         user_stack_formatted = "".join(
                             traceback.format_list(user_stack)
                         )
+                        user_stack_trace = (
+                            "Graph break: skip: from user code at:\n%s"
+                            % user_stack_formatted
+                        )
+                        torch._logging.trace_structured(
+                            "dynamo_graph_break_reason",
```

Comment:
This is just vanilla file handling in tlparse, so you can use the "artifact" event type to skip needing to make a tlparse change. Looks like this

```
        torch._logging.trace_structured(
            "artifact",
            metadata_fn=lambda: {
                "name": "fx_graph_cache_hash",
                "encoding": "json",
            },
            payload_fn=lambda: json.dumps(cache_info),
        )
```